src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,28 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String bulldozerModel;
```

Comment:
Don't forget about access modifiers for your custom fields in `Bulldozer` class. Fields should be private to follow the encapsulation principle.